frontend/ui/widget/multiconfirmbox.lua

```diff
@@ -48,6 +48,8 @@ local MultiConfirmBox = InputContainer:new{
     choice1_callback = function() end,
     choice2_callback = function() end,
     cancel_callback = function() end,
+    choice1_enabled = true,
```

Comment:
Looks like this doesn't support `text_func` atm? That's slightly unfortunate but it'd be easy to add. I assume the underlying Button(Table) already supports it.